signup.html

```diff
@@ -1,51 +1,86 @@
 <!DOCTYPE html>
 <html lang="ko">
   <head>
-    <meta charset="utf-8">
-    <title>회원가입</title>
-    <link rel="stylesheet" as="style" crossorigin href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css">
-    <link rel="stylesheet" href="styleLogin.css">
+    <meta charset="utf-8" />
+    <title>판다마켓-회원가입</title>
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.6/dist/web/static/pretendard.css"
+    />
+    <link rel="stylesheet" href="auth.css" />
+    <link rel="stylesheet" href="global.css" />
   </head>
   <body>
     <div class="content">
       <div class="head-img">
         <a class="login-link" href="/">
-          <img class="pandaMarketImg" src="img/logo.png">
+          <img class="pandaMarketImg" src="img/logo.png" />
         </a>
       </div>
-      <div>
-        <p>이메일</p> 
-        <input type="email" name="userEmail" id="userName">
-        <p>닉네임</p> 
-        <input type="text" name="userName" id="nicname">
-        <p>비밀번호</p>
-        <input type="password" name="userPassword" id="userPassword">
-        <p>비밀번호 확인</p>
-        <input type="password" name="userPassword" id="userPassword">
-      </div>
-      <div class="content-login">
-        <a class="login-button" href="/signup">회원가입</a>
-      </div>
+      <form action="/signup" method="post">
+        <div class="input-content">
+          <label for="userName" class="userName-label">이메일</label>
+          <input
+            type="email"
+            name="userEmail"
+            id="userName"
+            class="userName"
+            placeholder="이메일을 입력해주세요"
+          />
+          <label for="nicname" class="nicname-label">닉네임</label>
+          <input
+            type="text"
+            name="userName"
+            id="nicname"
+            class="nicname"
+            placeholder="닉네임을 입력해주세요"
+          />
+          <label for="userPassword" class="userPassword-label">비밀번호</label>
+          <input
+            type="password"
+            name="userPassword"
+            id="userPassword"
+            class="userPassword"
+            placeholder="비밀번호를 입력해주세요"
+          />
+          <label for="checkPassword" class="checkPassword-label"
+            >비밀번호 확인</label
+          >
```

Comment:
💊 제안
현재 코드에서는 하나의 태그가 나눠져 있어 가독성이 조금 떨어질 수 있습니다.  
태그 간 계층 구조를 명확하게 표현하기 위해 아래처럼 작성하시는 것을 추천드려요~

```suggestion
          <label for="checkPassword" class="checkPassword-label">
            비밀번호 확인
          </label>
```